gcore/tilematrixset.cpp

```diff
@@ -77,7 +77,12 @@ std::unique_ptr<TileMatrixSet> TileMatrixSet::parse(const char *fileOrDef)
     std::unique_ptr<TileMatrixSet> poTMS(new TileMatrixSet());
 
     constexpr double HALF_CIRCUMFERENCE = 6378137 * M_PI;
-    if (EQUAL(fileOrDef, "GoogleMapsCompatible"))
+    if (EQUAL(fileOrDef, "GoogleMapsCompatible") ||
+        EQUAL(fileOrDef, "http://www.opengis.net/def/tilematrixset/OGC/1.0/"
```

Comment:
@doublebyte1 `GoogleCRS84Quad` does not seem correct here.
That usually refers to the same TileMatrixSet as _WorldCRS84Quad_, except with the zoom levels offset by 1, with level 0 having the latitude going from -180..180, based on the _GoogleCRS84Quad_ Well-Known Scale Set specified in [WMTS 1.0 Annex E.3](https://portal.ogc.org/files/?artifact_id=35326):

https://maps.gnosis.earth/ogcapi/collections/blueMarble/map/tiles/GoogleCRS84Quad/0/0/0.jpg

![image](https://user-images.githubusercontent.com/5140074/235354503-3acd999c-d348-4383-907c-8ac2551d4517.png)

I'm not sure if GDAL pre-defines this TileMatrixSet, but this section here is for spherical mercator (EPSG:3857).

_GoogleCRS84Quad_ is in EPSG:4326 like the _InspireCRS84Quad_ below, except it has that extra lower resolution level.

Reply:
https://github.com/OSGeo/gdal/pull/7656/commits/5ae4b5948557678cfa7bb57a321a8922b4634903

Reply:
@doublebyte1 Yes that solves that issue.